job-utilities/job_router/lambda_function.py

```diff
@@ -19,36 +19,40 @@ def handler(event, _):
     CMR_LB_URL - The LB used for routing calls to CMR
     """
     environment = os.getenv('CMR_ENVIRONMENT')
-    cmr_url = os.getenv('CMR_LB_URL')
+    cmr_lb_name = os.getenv('CMR_LB_NAME')
     service = event.get('service', 'bootstrap')
     endpoint = event.get('endpoint')
     single_target = event.get('single-target', True)
-
-    error_state = False
+    request_type = event.get('request-type', "GET")
 
     if environment is None:
```

Comment:
Just curious: you are trying to avoid having multiple error exit points so that's why it's deferred? Otherwise you could just exit as soon as you tried and failed to get the environment

Reply:
Figured it was fine to do it this way with only a couple variables, but anything larger probably would've had it done one by one